src/display/api.js

```diff
@@ -299,6 +302,9 @@ PDFJS.getDocument = function getDocument(src,
     source = src;
   }
 
+  // Default range chunk size to 2^16 = 65536
+  source.rangeChunkSize = source.rangeChunkSize || 65536;
```

Comment:
As alternative, you can move the constant and default value replacement to the ChunkedStreamManager constructor.